### DISCUSSION TITLE
[BUG]-Apply-PnPProvisioningTemplate -Path on sharepoint online issue : Object reference not set to an instance of an object

### DISCUSSION THREAD

**Discussion by klouva76** [pnp/powershell#415] [General]
Hello, 

I am trying to provision a Modern SharePoint online, using Azure Automation based on PnpPowershell Runbook and I am facing and error when executing this command 
**Apply-PnPProvisioningTemplate -Path $varXmlFilePath**

The context : 
My xml file is stored in a document library on sharepoint online / or One drive and the execution of the runbook is giving this error : 
**Apply-PnPProvisioningTemplate : Object reference not set to an instance of an object**

NB: 
The cmdlet is working fine when I an using Local PowerShell console (Path parameter is locall....OK)

Thank you!

**Comment by gautamdsheth** [comment]
Hello :) 

How are you reading this file from the document library in Azure automation ?
What version of PnP are you using ?
This is a very generic error, can you execute `Get-PnPException` after the commandlet and see what error is logged ?
Moving this to discussion since this is not a bug as it works in your local environment.
